controllers/gce/loadbalancers/loadbalancers.go

```diff
@@ -367,7 +367,10 @@ func (l *L7) checkSSLCert() (err error) {
 	if l.sslCert != nil {
 		certName = l.sslCert.Name
 	}
-	cert, _ := l.cloud.GetSslCertificate(certName)
+	cert, err := l.cloud.GetSslCertificate(certName)
```

Comment:
this is because the cert couldn't exist, and that's an error, but we need to create it. If the cert did exist and we still get an error, then the create will err out and we'll retry, so eitherway we don't want to retry on GET error (which is what returning the error will do).